airbyte-ci/connectors/connector_ops/connector_ops/utils.py

```diff
@@ -272,9 +273,32 @@ def version_in_dockerfile_label(self) -> str:
     def name_from_metadata(self) -> Optional[str]:
         return self.metadata.get("name") if self.metadata else None
 
+
+    @property
+    def support_level(self) -> Optional[str]:
+        return self.metadata.get("supportLevel") if self.metadata else None
+
     @property
-    def release_stage(self) -> Optional[str]:
-        return self.metadata.get("releaseStage") if self.metadata else None
+    def ab_internal_sl(self) -> int:
+        default_value = 100
+        sl_value = get(self.metadata, "ab_internal.sl")
+
+        if sl_value is None:
+            logging.warning(f"Connector {self.technical_name} does not have a support level defined in metadata.yaml. Defaulting to {default_value}")
+            return default_value
+
+        return sl_value
+
+    @property
+    def ab_internal_ql(self) -> int:
```

Comment:
These should already be defined but putting defaults here in case.

These should be converted to hard errors on the introduction of metadata v2 where `ab_internal` is required